libs/barista-components/filter-field/src/types.ts

```diff
@@ -156,10 +157,11 @@ export function dtAutocompleteDef<D = unknown, OG = unknown, Op = unknown>(
   optionsOrGroups: DtNodeDef<OG>[],
   distinct: boolean,
   async: boolean,
+  partial: boolean,
```

Comment:
```suggestion
  partial: boolean = false,
```
We need to set a default value and by doing so make this optional, otherwise this would introduce a breaking change.